view/frontend/web/js/view/checkout/summary/normalize-cart.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Extend (c) 2022. All rights reserved.
```

Comment:
tiny inconsequential nit: could probably make this (c) 2023 (and in the html file as well)